tests/safeds_stubgen/stubs_generator/__snapshots__/test_generate_stubs/TestStubFileGeneration.test_stub_creation[class_module].sdsstub

```diff
@@ -1,5 +1,5 @@
-@PythonModule("tests.data.various_modules_package.class_module")
-package tests.data.variousModulesPackage.classModule
+@PythonModule("tests.data.various_modules_package")
+package tests.data.variousModulesPackage
```

Comment:
That does not seem right: Only `ClassModuleClassD` is reexported under the path `tests.data.various_modules_packages`. The other classes should still be in `tests.data.various_modules_packages.class_module`.